config/gocdb_schema.xml

```diff
@@ -163,6 +163,25 @@
 	    <length>255</length>
 	    <regex>/^(([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6}[,;]?)+$/</regex>
 	</field>
+  <field>
```

Comment:
This block seems differently indented to [this](https://github.com/GOCDB/gocdb/pull/161/files#diff-950b814119a26dd29c62aac47af704adR371) block (2 spaces versus 4 spaces) in the same file.

It will be easier if we keep the new indenting consistent with the old in the file.